src/goto-instrument/accelerate/scratch_program.h

```diff
@@ -77,8 +77,8 @@ class scratch_programt:public goto_programt
       symex(mh, symbol_table, equation, options, path_storage, guard_manager),
       satcheck(util_make_unique<satcheckt>(mh)),
       satchecker(ns, *satcheck, mh),
-      z3(ns, "accelerate", "", "", smt2_dect::solvert::Z3, mh),
-      checker(&z3) // checker(&satchecker)
+      solver(ns, "accelerate", "", "", smt2_dect::solvert::CPROVER_SMT2, mh),
```

Comment:
Do we know how this change of solver affects performance?

Reply:
Please ignore this change. I made a mistake in my local testing. This doesn't actually fix the error. @thomasspriggs made a separate change that fixes the root cause.